src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -4,12 +4,14 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Internal;
```

Comment:
It seems wrong for KestrelServer to depend on Hosting.Internal.  I would move ServerAddressesFeature to a public namespace.